plugins/catalog-backend/src/tests/migrations.test.ts

```diff
@@ -395,4 +395,79 @@ describe('migrations', () => {
       await knex.destroy();
     },
   );
+
+  it.each(databases.eachSupportedId())(
+    '20241024104700_add_entity_ref_to_final_entities.js, %p',
+    async databaseId => {
+      const knex = await databases.init(databaseId);
+
+      await migrateUntilBefore(
+        knex,
+        '20241024104700_add_entity_ref_to_final_entities.js',
+      );
+
+      await knex
+        .insert({
+          entity_id: '8222246a-b572-49cf-a702-1a0fcfaae901',
+          entity_ref: 'k:ns/n',
+          unprocessed_entity: '{}',
+          processed_entity: '{}',
+          errors: '[]',
+          next_update_at: knex.fn.now(),
+          last_discovery_at: knex.fn.now(),
+        })
+        .into('refresh_state');
+
+      // Insert a simple URL before the migration
+      await knex
+        .insert({
+          entity_id: '8222246a-b572-49cf-a702-1a0fcfaae901',
+          hash: '3f5a4d6ba8507be297bb7cd87c4b55b63e3f4c14',
+          stitch_ticket: '52367ed7-120b-405f-b7e0-cdd90f956312',
+          final_entity: '{}',
+        })
+        .into('final_entities');
+
+      await migrateUpOnce(knex);
+
+      // verify that the entity_ref column has been added
+      const columnInfo = await knex('final_entities').columnInfo();
```

Comment:
TIL!

can you do this exact same but inverse check before `migrateUpOnce`, just to demonstrate that it went from not defined to defined?